lisa/nic.py

```diff
@@ -358,67 +363,55 @@ def _get_nic_uuids(self) -> None:
 
     def _load_nics(self) -> None:
         # Identify which nics are slaved to master devices.
-        # This should be really simple with /usr/bin/ip but experience shows
-        # the tool isn't super consistent across distros in this regard
 
-        # use sysfs to gather synthetic/pci nic pairings and pci slot info
-        nic_info_fetch_cmd = "ls -la /sys/class/net/*/lower*/device"
         self._node.log.debug(f"Gathering NIC information on {self._node.name}.")
+        nic_names = self._get_nic_names()
```

Comment:
@LiliDeng please check if there is any regression.

Reply:
@squirrelsc  @LiliDeng 
If there hasn't been any regression could we merge and give this a shot this week? We can always back it out before including in the release candidate.